src/db/migrations/20250218094907_alter_and_add_profile_details_to_onerep_profiles_table.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export function up(knex) {
+  return knex.schema.alterTable("onerep_profiles", async (table) => {
+    table
+      .specificType("first_names", "character varying(255)[]")
+      .notNullable()
+      .defaultTo(knex.raw(`ARRAY[]::character varying(255)[]`));
+    table
+      .specificType("last_names", "character varying(255)[]")
+      .notNullable()
+      .defaultTo(knex.raw(`ARRAY[]::character varying(255)[]`));
+    table
+      .specificType("middle_names", "character varying(255)[]")
+      .notNullable()
+      .defaultTo(knex.raw(`ARRAY[]::character varying(255)[]`));
+    table
+      .specificType("phone_numbers", "character varying(255)[]")
+      .notNullable()
+      .defaultTo(knex.raw(`ARRAY[]::character varying(255)[]`));
+    table.jsonb("addresses").notNullable().defaultTo("[]");
+
+    await knex.raw(`
+      UPDATE "onerep_profiles"
+      SET "addresses" = jsonb_build_array(
+        jsonb_build_object(
+          'city', "city_name",
+          'state', "state_code"
+        )
```

Comment:
Shouldn't we also update the `first_names`, `middle_names`, etc. columns as well to copy the existing data?

Reply:
`first_name` and `middle_name` stay as separate columns and only alternate first and last names will be written to `first_names` and `middle_names`. That matches the schema for [OneReps profiles](https://docs.onerep.com/#operation/getProfile).

Reply:
Ah gotcha.